dev/breeze/src/airflow_breeze/commands/minor_release_command.py

```diff
@@ -69,7 +69,7 @@ def commit_changes(version_branch):
         if DRY_RUN:
             console_print("Skipping below command on CI")
         run_command(
-            ["git", "commit", "-m", f"Update default branches for {version_branch}"],
+            ["git", "commit", "-m", f"Update default branches for {version_branch}", "--no-verify"],
```

Comment:
Allowing this to run `pre-commit` gives room to updating hashes which we can do in a separate commit